include/boost/test/impl/framework.ipp

```diff
@@ -269,7 +269,7 @@ private:
         std::vector<component> const& filters = m_components[m_depth-1];
 
         // look for match
-        // using namespace boost::placeholders;
+        using namespace boost::placeholders;
```

Comment:
This matches with their move (not declaring `using namespace boost::placeholders` in global header. Then it needs to be done individual implementation file like this.

Reply:
How does this fix look?

Reply:
OK, does this match upstream?

Reply:
Yes, I'm not sure why it was different originally.